public/date-helpers.js

```diff
@@ -67,9 +67,28 @@ export function parseDateIntoLocalTimezone(s){
     return new Date(...ds);
 }
 
-export const DAY_IN_MS = 1000 * 60 * 60 * 24;
+/**
+ * Parse an 8601 date string `YYYY-MM-DD` into a date.
+ * @export
+ * @param {string} str
+ * @returns {Date}
+ */
+export function parseDate8601String(str){
+    // This should just work, we can get fancy later and use date-fns or something.
+    return new Date(str);
```

Comment:
I think there's some timezone adjustment I have to do with https://github.com/bitovi/jira-timeline-report/blob/main/public/date-helpers.js#L48